.github/Dockerfile

```diff
@@ -0,0 +1,77 @@
+FROM ubuntu:bionic
```

Comment:
Do you think that we should move this file to something like `.github/images/build/Dockerfile`?

I don't know if I like leaving it at the root of `.github` but maybe that's just a matter of personal taste :D